test/main_test.go

```diff
@@ -175,28 +175,38 @@ func deleteNamespaceIfExists(namespace string) {
 	})
 }
 
-func matchingFeature(tags string, features []*gherkin.Feature) bool {
+func matchingFeature(filterTags string, features []*feature) bool {
 	for _, ft := range features {
-		if len(getMatchingScenarios(tags, ft)) > 0 {
+		if matchesPickles(filterTags, ft.pickles) {
 			return true
 		}
 	}
 	return false
 }
 
-func showScenarios(features []*gherkin.Feature) {
+func matchesPickles(filterTags string, pickles []*messages.Pickle) bool {
+	for _, pickle := range pickles {
+		if matchesTags(filterTags, pickle.Tags) {
+			return true
+		}
+	}
+	return false
+}
+
+func showScenarios(features []*feature, showSteps bool) {
 	mainLogger := framework.GetMainLogger()
 	mainLogger.Info("------------------ SHOW SCENARIOS ------------------")
 	for _, ft := range features {
-		if len(ft.ScenarioDefinitions) > 0 {
-			mainLogger.Infof("Feature: %s", ft.Name)
-			for _, scenario := range ft.ScenarioDefinitions {
-				scenarioMessage := framework.GetScenarioName(scenario)
-
-				if scenarioOutline, ok := scenario.(*gherkin.ScenarioOutline); ok {
-					scenarioMessage += fmt.Sprintf(" (Examples: %s)", strings.Join(framework.GetExamplesNames(scenarioOutline), ", "))
+		// Placeholders in names are now replace directly into names for each pickle
```

Comment:
Ah, so it is a new feature in the end :)

BTW `are now replaceD`

Reply:
thx. updated